packages/artifact/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/artifact",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "preview": true,
```

Comment:
We should update all the package-lock.json as well, alongside release notes

Reply:
👍  I think CI is failing because I need to update the `package-lock.json`s.  I'm struggling a bit to figure out how to get `npm` to update them with `lerna` in the mix